QuizBowlDiscordScoreTracker/Commands/AdminCommandHandler.cs

```diff
@@ -26,9 +26,10 @@ public AdminCommandHandler(ICommandContext context, IDatabaseActionFactory dbAct
 
         private IDatabaseActionFactory DatabaseActionFactory { get; }
 
-        public async Task CheckPermissionsAsync()
+        public async Task CheckPermissionsAsync([Summary("Text channel mention (#textChannelName)")] IMessageChannel messageChannel)
```

Comment:
Arguments to Handler methods don't need the Summary field. Summary is only used on the Command classes, to build the message sent by !help